networking-calico/devstack/bootstrap.sh

```diff
@@ -163,11 +163,12 @@ ls -la /opt/stack
 
 # Stack!
 sudo -u stack -H -E bash -x <<'EOF'
-
-set
 cd /opt/stack/devstack
 ./stack.sh
+EOF
 
+sudo -u stack -H -E bash -x <<'EOF'
```

Comment:
Is it worth a little comment to explain why this is here twice?  It does look odd, but I understand from the explanation in the PR why its here :)